src/components/common/validations/EIKValidator.tsx

```diff
@@ -0,0 +1,83 @@
+export function calculateChecksumForNineDigitsEIK(eik: string): boolean {
```

Comment:
Can we put some link here as a comment of the actual documentation about the algorithm?

Reply:
I can't find a link to official documentation for the algorithm.

Reply:
> I can't find a link to official documentation for the algorithm.
[link](https://www.nsi.bg/sites/default/files/konkursi/RGP_OPAK_2014_Annex_6.pdf)

Reply:
> > I can't find a link to official documentation for the algorithm.
> 
> [link](https://www.nsi.bg/sites/default/files/konkursi/RGP_OPAK_2014_Annex_6.pdf)

I added this link